lib/Launcher.js

```diff
@@ -22,7 +22,7 @@ const Browser = require('./Browser');
 const readline = require('readline');
 const crypto = require('crypto');
 
-const CHROME_PROFILE_PATH = path.resolve(__dirname, '..', '.dev_profile');
+const CHROME_PROFILE_PATH = path.join(os.tmpdir(), 'puppeteer_dev_profile');
```

Comment:
does this work on windows? For some reason, chrome launcher has a bunch of heuristics to infer tmp directory